src/components/ProposalListGrid.tsx

```diff
@@ -10,8 +10,19 @@ interface ProposalListGridProps {
 export const ProposalListGrid = ({ proposals }: ProposalListGridProps) => {
   const { proposalId } = useParams();
 
+  const listGridRef = React.useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    // Scroll to the previous sibling to emphasize that there are (probably)
+    // previous elements in the list out of scroll-view.
+    listGridRef.current?.querySelector('.active')?.previousElementSibling?.scrollIntoView();
+  }, []);
```

Comment:
As written, this is a one-shot. I think that's probably the right call, but I wanted to check: should this run again if the currently selected proposal changes? Probably not, because clicking on a proposal shouldn't scroll the list out from under the mouse.

Reply:
That's correct! This scroll ensures the current proposal is visible in the sidebar _when the page renders_ to give the user a sense of geography. After that the user should feel entirely in control. Clicking a new proposal doesn't need to re-scroll the sidebar, because the user's sense of geography isn't at risk.

Reply:
Well stated! Thanks for talking through that with me.